docs/api-docs/users/Account-level-API-accounts.mdx

```diff
@@ -0,0 +1,52 @@
+# Account-level API accounts
+
+## Introduction
+The Account-Level API is designed to allow merchants to build custom integrations and solutions that require high-level access to a store's resources. By using this API, merchants can automate various business need processes, integrate with third-party applications, and manage multiple stores per account.
+
+<Callout type="info">
+    When a token is generated it will belong to the account that the merchant has logged into.
+</Callout> 
+
+With Account-Level API credentials merchants can now perform actions such as:
+* Create tokens that can be used to manage apps on a merchant's store
+* Manage charges and subscriptions for accounts
+* Integrate third-party applications to streamline and automate store processes
+
+<Callout type="info">
+    As long as this account is active, the token will always be tied to the account.
+</Callout>
+
+Before you get started creating Account-Level tokens, read our doc on [API Accounts & OAuth Scopes](api-docs/getting-started/api-accounts) to get up to speed.
+
+## Prerequisites
+
+Head into action with Account-level API management by verifying that you have the following prerequisites.
+
+* A BigCommerce store
+* A valid API account access token with the necessary credentials to make these changes
+
+## OAuth scopes
```

Comment:
@bc-tgomez I believe these need to be added to the [OAuth scopes article](https://developer.bigcommerce.com/api-docs/getting-started/api-accounts#oauth-scopes)?

Reply:
Here's the [new ticket](https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4868) for reference.